src/ToDo/Business/TaskService.cs

```diff
@@ -69,4 +69,44 @@ public async ValueTask<IImmutableList<ToDoTask>> GetAllAsync(string displayName
 			.Select(data => new ToDoTask(data.ParentList!.Id!, data))
 			.ToImmutableList();
 	}
+
+	/// <inheritdoc />
+
+	public async Task AddStepAsync(string baseTaskListId, string baseTaskId,
+		CheckListItem checkListItem, CancellationToken ct)
+	{
+		var createdStep = await _client.AddStepAsync(baseTaskListId, baseTaskId, checkListItem.ToData(), ct);
+
+		//TODO: add messenger notification
```

Comment:
Do you have an issue/item in the backlog for the 3 To-Do(s) you added in TaskService  ?

Reply:
It's only a Task, service layer for task step